### PR TITLE
fix(retry): add default behavior about retry

### PR DESCRIPTION
**Pull request by davinkevin**
Closes https://github.com/istio/istio/issues/20547

### CODE REVIEW

File: content/en/docs/concepts/traffic-management/index.md [hunk 1]
```diff
@@ -666,8 +666,8 @@ availability and application performance by making sure that calls don’t fail
 permanently because of transient problems such as a temporarily overloaded
 service or network. The interval between retries (25ms+) is variable and
 determined automatically by Istio, preventing the called service from being
-overwhelmed with requests. By default, the Envoy proxy doesn’t attempt to
-reconnect to services after a first failure.
+overwhelmed with requests. The default retry behavior for HTTP requests is to
```
Comment (frankbu) [inline content/en/docs/concepts/traffic-management/index.md:669]:
```suggestion
overwhelmed with requests. The default retry behavior for HTTP requests is to
```

File: content/pt-br/docs/concepts/traffic-management/index.md [hunk 2]
```diff
@@ -674,8 +674,8 @@ availability and application performance by making sure that calls don’t fail
 permanently because of transient problems such as a temporarily overloaded
 service or network. The interval between retries (25ms+) is variable and
 determined automatically by Istio, preventing the called service from being
-overwhelmed with requests. By default, the Envoy proxy doesn’t attempt to
-reconnect to services after a first failure.
+overwhelmed with requests. The default retry behavior for HTTP requests is to
```
Comment (frankbu) [inline content/pt-br/docs/concepts/traffic-management/index.md:677]:
```suggestion
overwhelmed with requests. The default retry behavior for HTTP requests is to
```

**Comment by frankbu** [comment]
@davinkevin Please remove the trailing spaces causing the linter to fail.

**Comment by davinkevin** [comment]
> @davinkevin Please remove the trailing spaces causing the linter to fail.

Modification done. Sorry about that 😉.

File: content/en/docs/concepts/traffic-management/index.md [hunk 3]
```diff
@@ -666,8 +666,8 @@ availability and application performance by making sure that calls don’t fail
...
+retry a call twice before sending back the error.
```
Comment (frankbu) [inline content/en/docs/concepts/traffic-management/index.md:670]: Maybe the following wording would be a little better?
```suggestion
retry twice before returning the error.
```

**Pull request merged**